Bottom: 8b3e917b342f04315141dc9cff278de436f6888d
Top:    73a7bdcd035349ed13f97d56fdb9448a2f694569
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-16 21:16:19 -0700

Refresh of librdmacm-5-8-configure-use-au

---

diff --git a/autogen.sh b/autogen.sh
index 6c9233e..24c0a76 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,4 +1,5 @@
 #! /bin/sh
 
 set -x
+test -d ./config || mkdir ./config
 autoreconf -ifv -I config
